Mono.Debugging/Mono.Debugging.Client/DebuggerSession.cs

```diff
@@ -410,9 +410,10 @@ public void Run (DebuggerStartInfo startInfo, DebuggerSessionOptions options)
 			if (options == null)
 				throw new ArgumentNullException (nameof (options));
 
+			Interlocked.Exchange (ref this.options, options);
```

Comment:
Doesn't AttachToProcess need the same change?

Reply:
Yeah, it does, it was also changed in the previous PR

Reply:
Thanks, fixed